### PR TITLE
Missed Kurdish language dialects

### PR DESCRIPTION
**Pull request by halwesit**
The Kurdish language has 2 main dialects (Sorani - Iraq, Kurmanji - Turkey) that are so different based on the typography and the meaning of the words. for example the Kurdish Sorani is an RTL language, based on the Arabic script. The Kurdish Kurmanji is LTR, based on the Latin script.

## Description

Fixes #

## Type of Change

- [ ] Bugfix
- [x] Improvement
- [ ] New Feature
- [ ] Refactor / codestyle updates
- [ ] Other, please describe:

## Requirements Checklist

- [ ] New / updated tests are included
- [ ] All tests are passing locally
- [ ] Performed a self-review of the submitted code

If adding a new feature:

- [ ] Documentation was added/updated

### CODE REVIEW

**Comment by rijkvanzanten** [comment]
@halwesit Are you planning on providing the translations for these dialects as well? I'm a tad worried that this will add the language to the platform, but that nobody will actually contribute the correct translations 🤔 The current Kurdish (which I believe is Sorani) is only at 4%, so I'm a little worried this will add another untranslated language 👀

**Comment by halwesit** [comment]
Actually, my dialect is Sorani Kurdish ku-IQ, and that 4% was translated by me, I did it because I thought it is Sorani Kurdish, but when I realized that its for Kurmanji Kurdish ku-TR, I stopped doing it. 
I'm unable to translate Kurdish Kurmanji ku-IQ because it's far different from my dialect Kurdish Sorani.

**Comment by rijkvanzanten** [comment]
Ah gotcha! I've updated Crowdin (https://locales.directus.io/project/directus) to now show Kurmanji and Sorani separate 🙂 Thanks!

**Comment by halwesit** [comment]
thank you so much.

File: app/src/lang/available-languages.yaml [hunk 1]
```diff
@@ -28,6 +28,8 @@ id-ID: Indonesian (Indonesia)
 it-IT: Italian (Italy)
 ja-JP: Japanese (Japan)
 ko-KR: Korean (Korea)
+ku-TR: Kurdish Kurmanji (Turkey)
+ku-IQ: Kurdish Sorani (Iraq)
```
Comment (rijkvanzanten) [inline app/src/lang/available-languages.yaml:32]:
```suggestion
kmr-TR: Kurdish Kurmanji (Turkey)
ckb-IR: Kurdish Sorani (Iraq)
```

**Review by rijkvanzanten — CHANGES_REQUESTED** [review]

**Comment by rijkvanzanten** [comment]
I've updated the locale codes to match what our translations provider relies on: https://github.com/directus/directus/pull/14889/files

**Review by rijkvanzanten — APPROVED** [review]

**Comment by roj1512** [comment]
@halwesit These codes are not very appealing. I Kurdish should either not have a country code.

**Pull request merged**